python/pc_ble_driver_py/ble_driver.py

```diff
@@ -361,6 +361,8 @@ def from_c(cls, adv_report_evt):
         index           = 0
         while index < len(ad_list):
             ad_len  = ad_list[index]
+            if ad_len == 0:
+                break
```

Comment:
Yes, I was thinking that also.

But the current IndexError exception is "logger.error('Invalid advertising data' ...). Is having zero length advdata field "invalid"?

Also, it might be valid to have zero-length field followed by more valid fields, so my break solution is not a good solution.